### DISCUSSION TITLE
How to Serialization & Deserialization for caption in Image

### DISCUSSION THREAD

**Discussion by duypham90** [facebook/lexical#4730] [Q&A]
I don't know how to get  all caption in image after Editor change.
Current I only get image but don't have caption
Please help me or guide me some thing how to get image caption from Editor.
Thanks

![image](https://github.com/facebook/lexical/assets/24804332/4dd976ed-1e21-47a8-a8f7-b931317e8407)

![image](https://github.com/facebook/lexical/assets/24804332/177f28ff-5ede-40c3-9ea7-5ae2cd8436b3)
I edit the file below as as new to get Html from Lexical editor

packages\lexical-playground\src\Editor.tsx

```typescript
  useEffect(() => {
    editor.registerUpdateListener(() => {
      editor.update(() => {
        const html = $generateHtmlFromNodes(editor, null);
        console.log(html);
      });
    });
  }, [editor]);
```
The result:

![image](https://github.com/facebook/lexical/assets/24804332/0e1dbeb8-1185-4de1-8f28-9ede208124e4)

**Comment by duyphamsalto** [comment]
Can you help me !!!
